corehq/apps/tour/templates/tour/config/new_blank_app.html

```diff
@@ -0,0 +1,63 @@
+{% load i18n %}
+<script type="text/javascript">
```

Comment:
sorry, yeah, was just pinging the wider team in case any clever thoughts (i didn't have any) but wasn't suggesting this was a blocker